File: .vscode/IFAL/C/[#6].c
// [#6] Leia três números e que imprima o maior e o menor.

#include <stdio.h>
int main()
{
    float n1, n2, n3;

    printf("Qual o primeiro número? ");
    scanf("%f", &n1);
    printf("Qual o segundo número? ");
    scanf("%f", &n2);
    printf("Qual o terceiro número?  ");
    scanf("%f", &n3);

    if (n1 > n2 && n1 > n3){
   	 printf("\nO maior número é %.0f", n1);

    } else if (n2 > n1 && n2 > n3){
   	 printf("\nO maior número é %.0f", n2);

    } else{
   	 printf("\nO maior número é %0.f", n3);
    }

    return 0;
}

File: programas_python/911/[#6].py
'''[#6] Leia três números e que imprima o maior e o menor.'''

n1 = float(input("Qual o primeiro número? "))
n2 = float(input("Qual o segundo número? "))
n3 = float(input("Qual o terceiro número? "))

n4 = sorted([n1, n2, n3])

print("O maior número é {:.0f} e o menor número é {:.0f}".format(n4[2],n4[0]))